test/image/mocks/zz-pie-slice-legend.json

```
@@ -0,0 +1,50 @@
+{
+ "data": [
+  {
+   "labels": [
+     "with",
+     "pie",
+     "multiple",
+     "hidden",
+     "legends",
+     "A",
+    "hidden too"
+    ],
+   "values": [3, 4, 2, 7, 1, 5, 8],
+   "type": "pie",
+   "showlegend": [true, true, true, false, true, true, false],
+   "legend": ["legend2", "legend", "legend2", "legend4", "legend3", "legend", "legend4"]
+  }
+ ],
+ "layout": {
+  "title": {
+    "text": "Test array version of 'showlegend' and 'legend'"
+  },
+  "width": 400,
```

Could you increase the width to avoid the slice label getting covered by the legend?

```suggestion
  "width": 500,
```